include/OpenSpaceToolkit/Astrodynamics/Trajectory/Models/Tabulated.hpp

```diff
@@ -48,7 +48,7 @@ using ostk::physics::time::Scale;
 using ostk::astro::trajectory::Model;
 using ostk::astro::trajectory::State;
 
-#define DEFAULT_TABULATED_INTERPOLATION_TYPE Tabulated::InterpolationType::Linear
+#define DEFAULT_TABULATED_INTERPOLATION_TYPE Tabulated::InterpolationType::Linear  // TBM: once OSTk math MR is merged
```

Comment:
Looks like we can remove this now?
Can do it in a follow up MR.

Reply:
Yup forgot this one
```suggestion

```